doc/dev/index.rst

```diff
@@ -841,6 +865,11 @@ An individual test from the ceph-disk suite can be run by adding the
       --suite ceph-disk/basic \
       --filter 'ceph-disk/basic/{distros/ubuntu_14.04.yaml tasks/ceph-disk.yaml}'
 
+.. note: To run a standalone test, ``--suite`` alone is sufficient. If you
+   want to run a single combination out of a suite that is defined as a
+   directory tree, ``--suite`` must be combined with ``--filter``. This is
+   because the ``--suite`` option understands POSIX relative paths only.
```

Comment:
Hum, now I understand what you're suggesting. An example would be good to illustrate that and avoid confusion.